lib/ci-config-stack.ts

```diff
@@ -21,13 +21,16 @@ export class CIConfigStack extends Stack {
 
     static readonly REDIRECT_URL_SECRET_EXPORT_VALUE: string = 'redirectUrlSecret';
 
+    static readonly REDIRECT_OIDC_CLIENT_ID_SECRET_EXPORT_VALUE: string = 'OIDCClientIdSecret';
```

Comment:
The string value is cased differently than than the other variables, please keep them consistent, it would be odicClientIdSecret.  Why is this called `REDIRECT` it instead of something like OIDCConfigurationValues?